pkg/crc/machine/start.go

```diff
@@ -410,7 +410,7 @@ func (client *client) Start(ctx context.Context, startConfig StartConfig) (*Star
 
 	logging.Info("Adding crc-admin and crc-developer contexts to kubeconfig...")
 	if err := writeKubeconfig(instanceIP, clusterConfig); err != nil {
-		logging.Warnf("Cannot update kubeconfig: %v", err)
+		logging.Errorf("Cannot update kubeconfig: %v", err)
```

Comment:
Slightly inaccurate commit log maybe, it says:
> The previous call ensures all operators are running

but when `WaitForClusterStable` fails, it only logs an error, it does not return, so we could perfectly get there without all operators running. But this is minor, makes sense to try to do these updates even if the operators are not all running.